### ISSUE TITLE
[4.6] Main window does not appear

### ISSUE THREAD

**Issue by yozhic** [pjheslin/diogenes#88]
The main window does not appear when running version 4.6 on Mac OS 10.14.6 Mojave (Build 18G103).

The new 4.6 version was installed by updating over the previous 4.5 version.

Calling the `Navigate` → `Home page` menu item fails:

```
Uncaught Exception:
TypeError: Cannot read properties of undefined (reading 'loadURL')
    at click (/Applications/Diogenes.app/Contents/Resources/app/main.js:571:26)
    at MenuItem.click (node:electron/js2c/browser_init:73:1908)
    at a._executeCommand (node:electron/js2c/browser_init:81:2453)
```

Calling the `Navigate` → `Reload app` menu item fails:

```
Uncaught Exception:
TypeError: Cannot read properties of undefined (reading 'webContents')
    at click (/Applications/Diogenes.app/Contents/Resources/app/main.js:585:24)
    at MenuItem.click (node:electron/js2c/browser_init:73:1908)
    at a._executeCommand (node:electron/js2c/browser_init:81:2453)
```

`Quit Diogenes` command fails:

```
Uncaught Exception:
Error: ENOENT: no such file or directory, unlink '/Users/yozhic/Library/Application Support/Diogenes/diogenes-lock.json'
    at Object.unlinkSync (node:fs:1735:3)
    at App.<anonymous> (/Applications/Diogenes.app/Contents/Resources/app/main.js:265:5)
    at App.emit (node:events:527:28)
```

Previous version 4.5 works fine on Mac OS 10.14.6, with no errors.

**Comment by pjheslin** [comment]
Thanks for the prompt and helpful report!  I tested the new version of Diogenes on two different Macbooks running Monterey before releasing it last night, and it works there.  But I have pulled out my old Mac running High Sierra and I was able to reproduce this bug.  It's the same problem as issue #89.  I thought the JSON.pm module was ubiquitous in Perl distributions, but apparently it's a relatively recent addition.  I'll work on fixing this today.
